main/main.go

```diff
@@ -134,7 +134,7 @@ func main() {
 		VerifierProtocol:              protocol,
 		RequestHash:                   client.RequestHash,
 		RequestPublicKeys:             client.RequestPublicKeys,
-		VerifyFromKnownIdentitiesOnly: false, // TODO: make configurable
+		VerifyFromKnownIdentitiesOnly: conf.VerifyFromKnownIdentitiesOnly,
 		VerificationTimeout:           time.Duration(conf.VerificationTimeoutMs) * time.Millisecond,
 	}
 
```

Comment:
It might be helpful to print the known internal and external UUIDs before server startup so users can be sure which UUIDs are allowed, especially in combination with VerifyFromKnownIdentitiesOnly=true. (Already done for internal identities in InitIdentities(conf.Devices) above but seemingly not for external IDs so far.)